tasks/changelog.rake

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+autoload :Changelog, "#{__dir__}/changelog"
+
+namespace :cl do
```

Comment:
I think it's better to name this `:changelog` for clarity's sake. It's going to be easier to discover this way.